src/sql/src/plan/expr.rs

```diff
@@ -1905,6 +1905,13 @@ impl HirRelationExpr {
 
         result
     }
+
+    /// True iff the expression contains a [`UnmaterializableFunc::MzNow`] call.
```

Comment:
```suggestion
    /// Whether the expression contains a [`UnmaterializableFunc::MzNow`] call.
```

We use this pattern elsewhere. Light preference to use here as well.